lib/spec_rewrite.ts

```diff
@@ -70,10 +76,11 @@ function vegaNonTransformToSql(tableName: string, markFields: string[]) {
   return `'${out}'`;
 }
 
-export const aggregateTransformToSql = (tableName: string, transform: AggregateTransform, db: string) => {
+export const aggregateTransformToSql = (tableName: string, transform: any, db: string, prev: any) => {
```

Comment:
Why any?

Reply:
@JunranY, could you take a look at this ?

Reply:
This is a change in this pull request so I think you could revert it and see what happens.

Reply:
I reverted it through my last pr https://github.com/leibatt/scalable-vega/pull/20 (made before this one, but committed later)

Reply:
@SaiArrow then you can just merge master into this